benchmark/src/rest-routing/routing-table.ts

```diff
@@ -75,7 +76,7 @@ function givenRouter(router: RestRouter, spec: OpenApiSpec, count: number) {
     log('Creating %s, %d', name, count);
     let start = process.hrtime();
 
-    const table = new RoutingTable(router);
+    const table = new HttpHandler(new Context(), router);
```

Comment:
nitpick: maybe rename `table` to `handler` as well? since it's not a routing table anymore.